### PR TITLE
wix-ui-core - <SigningPad />

### PR DESCRIPTION
**Pull request by giladsegal**
Add initial implementation for <SigningPad />

### CODE REVIEW

File: packages/wix-ui-core/src/components/signing-pad/test/SigningPad.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  it('should set data-hook prop', async () => {
+    const driver = createDriver(<SigningPad data-hook="comp" />);
+    const dataHook = await driver.getDataHook();
+    expect(dataHook).toBe('comp');
+  });
+
+  it('should call pad callback prop', async () => {
+    const padSpy = jest.fn();
+    createDriver(<SigningPad pad={padSpy} />);
+    expect(padSpy).toHaveBeenCalled();
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/components/signing-pad/test/SigningPad.spec.tsx]: Maybe - `toHaveBeenCalledWith()` and check instaceof HTMLCanvas?

File: packages/wix-ui-core/src/components/signature-input/test/SignatureInput.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  it('should invoke onClear callback', async () => {
+    const onClearSpy = jest.fn();
+    const driver = createDriver({ onClear: onClearSpy });
+    const clearButton = await driver.getChildDriverByHook(
+      TEST_IDS.CLEAR_BUTTON,
+    );
+    await clearButton.click();
+    expect(onClearSpy).toHaveBeenCalled();
+  });
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/components/signature-input/test/SignatureInput.spec.tsx]: Missing tests IMO - what happens when I pass wrong children (string/jsx) for clear button/title ? 
(Maybe adding propTypes?) - this library is sometimes consumed in JS.

File: packages/wix-ui-core/src/testkit/protractor.ts [hunk 3]
```diff
@@ -238,3 +238,12 @@ export const filePickerButtonTestkitFactory = protractorUniTestkitFactoryCreator
...
 export { FilePickerButtonUniDriver };
+
+import {
+  signaturePadUniDriverFactory,
+  SignatureInputDriver,
+} from '../components/signature-input/SignatureInput.uni.driver';
+export const signatureInputTestkitFactory = protractorUniTestkitFactoryCreator(
+  signaturePadUniDriverFactory,
+);
+export { SignatureInputDriver };
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/testkit/protractor.ts]: If you created a `UniDriver`, export it in all other testkits

File: packages/wix-ui-core/package.json [hunk 4]
```diff
@@ -92,6 +93,7 @@
     "@types/react-dom": "^16.0.5",
     "@types/react-onclickoutside": "^6.7.1",
     "@types/react-transition-group": "^2.0.6",
+    "@types/storybook__react": "^4.0.2",
```
Comment (NitayRabi) [inline packages/wix-ui-core/package.json]: This was missing? Walla?

File: packages/wix-ui-core/src/components/signature-input/test/SignatureInput.spec.tsx (hunk 2 above)
Comment (giladsegal) [reply packages/wix-ui-core/src/components/signature-input/test/SignatureInput.spec.tsx]: Done - can you have a look?

**Pull request merged**